### ISSUE TITLE
Captions with formatting can produce nested spans

### ISSUE THREAD

**Issue by friday** [sampotts/plyr#1003]
The `getCueAsHTML()` method converts cue markup to html. For example:

```xml
<v John Green>The human tongue is like <i.foreignphrase><lang ja>wasabi</lang></i>: it's very powerful, and should be used sparingly</v>
```

This will be converted to:

```html
<span title="John Green">The human tongue is like <i class="foreignphrase"><span lang="ja">wasabi</span></i>: it's very powerful, and should be used sparingly</span>
```

`<span>` can be added for metadata, but since Plyr uses span for styling the cue directly this causes issues.

![screen shot 2018-06-05 at 14 58 16](https://user-images.githubusercontent.com/515120/40977330-f5cc55c4-68d0-11e8-9a43-9bb7850c2155.png)

(Opacity has been increased to better illustrate the issue)

It happens in both Chrome and Firefox (haven't tested it in any other browser).

**Comment by friday** [comment]
Proposed solution:

Add a more specific selector for captions (or strip the html, but it's perhaps better as an option).

**Comment by sampotts** [comment]
Yeah I think we'd just need to add a `>` child combinator.

**Comment by friday** [comment]
That should work fine too.

**Comment by sampotts** [comment]
An additional selector would be cleaner though I suppose. `plyr__caption` or something.

**Comment by friday** [comment]
I'll fix this after #1000. Already have some other related changes locally.

**Comment by sampotts** [comment]
👍
